scripts/create_newcase

```diff
@@ -47,9 +47,15 @@ OR
                         help="Specify a compiler. "
                         "To see list of supported compilers for each machine, use the utility query_config in this directory")
 
+    parser.add_argument("--ncouplers",default=1,
+                        help="Specify number of coupler instances. "
+                        "Set the number of coupler instances in the case. "
+                        "If ncouplers is > 1 then ninst should = 1.")
+
     parser.add_argument("--ninst",default=1,
-                        help="Specify number of component instances"
-                        "Set the number of component instances in the case.")
+                        help="Specify number of component instances. "
+                        "Set the number of component instances in the case. "
+                        "If ninst > 1 then ncouplers should = 1")
```

Comment:
If ninst is > 1, then ncouplers must == 1 and visa versa, this is implemented here.   ninst is not a new argument to create_newcase, it's on the trunk.   You can still change it after the case is created.  Happy to work on documentation with you but this is not a new change in this branch, the only change here is the addition of -ncouplers and its analogous to how ninst was already implemented here.